test('Our first test', () => {

});

test('Our second test', () => {
    throw new Error("Error");
});